shuttle/db/db_functions.py

```diff
@@ -367,3 +395,13 @@ def get_db_locations():
     sql = "Select DISTINCT LOCATION from SHUTTLE_SCHEDULE"
     results = query(sql, 'read')
     return results
+
+
+# Called as a cronjob that clears every active request every night
+def clear_db_waitlist():
```

Comment:
could probably remove "db" from this since it isn't really needed and without it everything is understandable